rest_framework-stubs/test.pyi

```diff
@@ -14,6 +15,8 @@ from rest_framework.authtoken.models import Token
 from rest_framework.request import Request
 from rest_framework.response import _MonkeyPatchedResponse
 
+_GetDataType: TypeAlias = dict[str, Any] | str | list[tuple[str, str]] | None
```

Comment:
Hmm. `str` was an allowed input before, but looking at the code `django.utils.http.urlencode`, passing `str` to `APIRequestFactory.get` seems it will just crash? Unless I'm missing something

And the value is not just `str`, it can be `str | bytes | Iterable[str | bytes]`

And in function inputs, abstract base classes like `Iterable[]` or `Sequence[]` are preferred instead of `list[]`

```suggestion
_GetDataType: TypeAlias = Mapping[str, str | bytes | Iterable[str | bytes]] | Iterable[tuple[str, str | bytes | Iterable[str | bytes]]] | None
```